src/components/Toaster/index.tsx

```diff
@@ -122,6 +149,11 @@ const ToastContainer = (): JSX.Element => {
             toastClassName={localCss(styles.toast(theme))}
             autoClose={AUTOCLOSE_DELAY}
             icon={false}
+            newestOnTop={newestOnTop}
+            limit={limit}
+            className={className}
+            position={position}
+            containerId={id}
```

Comment:
should be sent by toast rather ?

Reply:
@chambo-e it's the container that manage all those settings

Reply:
ContainerId is only there if we want multiple Container (which I assume we don't :))
https://fkhadra.github.io/react-toastify/multi-containers

If we want uniq toast we need to use toastId on the toast itself https://fkhadra.github.io/react-toastify/use-a-custom-id